<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TianLiQtCommon_TimeLineLabel</class>
 <widget class="QWidget" name="TianLiQtCommon_TimeLineLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>31</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>31</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>31</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TianLiQtCommon_TimeLineLabel</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>0</y>
     <width>258</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HYWenHei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 13px;

color: #000000;</string>
   </property>
   <property name="text">
    <string>步骤</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProgressBackground">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>21</y>
     <width>258</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
background-color:rgba(0,0,0,26);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProgressBar">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>21</y>
     <width>10</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
background-color:rgb(253, 77, 77);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
